src/DB.php

```diff
@@ -758,6 +766,7 @@ public static function insertMessageRequest(Message &$message)
             $sth->bindParam(':message_id', $message_id, \PDO::PARAM_INT);
             $sth->bindParam(':user_id', $from_id, \PDO::PARAM_INT);
             $sth->bindParam(':date', $date, \PDO::PARAM_STR);
+            $sth->bindParam(':edit_date', $edit_date, \PDO::PARAM_STR);
```

Comment:
You're right about the ```$edit_date```, will adjust code to your suggestion.

```insertEditedMessageRequest()``` currently applies only to edited messages by the user

The changes I made in the code are specifically to update bot's message contaning callback query inline buttons (full message object is sent with the callback query)
edited messages that were pasted from inline mode are using ```insertEditedMessageRequest()``` (they are in fact posted by user - 'xxx via superbot' or something like that)

At first I wanted to insert every message edited by the bot with ```insertEditedMessageRequest()``` but couldn't find a way to do that, any ideas?